client/state/reader/follows/selectors/get-reader-followed-sites.js

```diff
@@ -19,7 +19,9 @@ const getReaderFollowedSites = createSelector(
 	( state ) => {
 		// remove subs where the sub has an error
 		return sortBy(
-			values( state.reader.follows.items ).filter( ( blog ) => blog.organization_id === NO_ORG_ID ),
```

Comment:
Same thing here as the notices one, not sure how to audit this.

Reply:
It's always an object, it defaults to an empty object at the reducer default state: https://github.com/Automattic/wp-calypso/blob/trunk/client/state/reader/follows/reducer.js#L111